docs/source/technical_tutorials/access/direct_access.ipynb

```diff
@@ -0,0 +1,1564 @@
+{
```

Comment:
<p>Which image did you test with? Can we test with the Pangeo image, this list can remain but Pangeo should have all these already.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I just use the basic vanilla workspace</p>

Reply:
<p>Go ahead and state that.</p><p>Also should we change this to <code>conda</code></p><p> <span class="ql-cursor">﻿</span></p>

Reply:
I thought this would be simple, swapping out the commands but it seems like the conda install isn't quite as straightforward - tested on the Pangeo workspace image. The vanilla workspace image errors when trying to set the libmamba solver, could be an issue with conda versions

Keeping this as pip for now and when conda versions get updated we can circle back and test/update all of the docs with conda install